oteapi_dlite/strategies/convert.py

```diff
@@ -115,6 +115,12 @@ class DLiteConvertStrategyConfig(AttrDict):
             description="Output instances.",
         ),
     ] = []
+    options: Annotated[
+        Optional[dict],
+        Field(
+            description="Extra options",
```

Comment:
Could we elaborate little here? What is the purpose of this configuration? Are these options sent to the underlying DLite driver?